code/modules/chemistry/Chemistry-Recipes.dm

```diff
@@ -1993,6 +1993,153 @@
 		mix_sound = 'sound/misc/drinkfizz.ogg'
 		drinkrecipe = TRUE
 
+	cola/cola2
+		required_reagents = list("cola_syrup" = 1, "sodawater" = 5)
+		result_amount = 6
+
+	cola_syrup
+		name = "Cola syrup"
+		id = "cola_syrup"
+		result = "cola_syrup"
+		required_reagents = list("cola" = 4, "VHFCS" = 1, "plasma" = 1)
+		result_amount = 2
+		min_temperature = 360
+		mix_phrase = "The cola simmers into a noxious black syrup."
+		instant = FALSE
+		reaction_speed = 2
+
+		on_reaction(datum/reagents/holder, created_volume)
+			. = ..()
+			var/datum/effects/system/harmless_smoke_spread/smoke = new /datum/effects/system/harmless_smoke_spread()
+			smoke.set_up(1, 0, get_turf(src))
+			smoke.start()
+
+	powercola
+		name = "Power Cola"
+		id = "powercola"
+		result = "powercola"
+		required_reagents = list("cola" = 1, "cola_syrup" = 1)
+		result_amount = 1
+		mix_phrase = "The cola grows more powerful than you could possibly imagine."
+		mix_sound = 'sound/misc/drinkfizz.ogg'
+		inhibitors = list("plasma")
+		drinkrecipe = TRUE
+
+	colamara_chaos
+		name = "Colamara Chaos"
+		id = "colamara"
+		result = "colamara"
+		required_reagents = list("powercola" = 1, "ice" = 0, "ldmatter" = 1, "bitters" = 1)
+		result_amount = 3
+		mix_phrase = "The bitters and cola roughen the surface of the ice as they war with one another."
+		mix_sound = 'sound/misc/drinkfizz.ogg'
+		drinkrecipe = TRUE
+
+	roaring_waters
+		name = "Roaring Waters"
+		id = "roaringwaters"
+		result = "roaringwaters"
+		required_reagents = list("powercola" = 1, "ice" = 0, "tonic" = 5, "sodawater" = 5)
+		result_amount = 10
+		mix_sound = 'sound/misc/drinkfizz.ogg'
+		mix_phrase = "A cool mist forms over the container as the waters crash over the ice."
+		drinkrecipe = TRUE
+
+	lostcoke
+		name = "The Lost Treasure of Kalimero"
+		id = "lostcoke"
+		result = "lostcoke"
+		required_reagents = list("powercola" = 1, "port" = 1)
+		result_amount = 2
+		mix_phrase = null
+		mix_sound = FALSE
+		drinkrecipe = TRUE
+
+		does_react(var/datum/reagents/holder)
+			var/atom/my_atom = holder.my_atom
+
+			if(length(holder.reagent_list) > length(src.required_reagents)) // drink must be pure to mix-- no teleporting chem bombs
+				return FALSE
```

Comment:
```suggestion
```
Okay but teleporting chem bombs randomly does sound funny and not all that dangerous. Blame me if we have to nerf this later :)

Reply:
I can logically reason out that making this possible would be more interesting for the game as a whole, but I don't really *want* this drink to be used as a teleporting chem bomb... it feels out of flavor. You could use chems like chocolate and smoke powder to trivialize the search process....